modules/op-installing-pipelines-as-code-cli.adoc

```diff
@@ -4,29 +4,29 @@
 
 :_content-type: PROCEDURE
 [id="installing-pipelines-as-code-cli_{context}"]
-= Installing {pac} CLI 
+= Installing {pac} CLI
 
 [role="_abstract"]
-Cluster administrators can use the `tkn-pac` CLI tool on local machines or as containers for testing. The `tkn-pac` CLI tool is installed automatically when you install the `tkn` CLI for {pipelines-title}.
+Cluster administrators can use the `tkn-pac` and `opc` CLI tools on local machines or as containers for testing. The `tkn-pac` and `opc` CLI tools are installed automatically when you install the `tkn` CLI for {pipelines-title}.
 
-You can also install the `tkn-pac` `tkn-pac` version `0.23.1` binaries for the supported platforms:
+You can install the `tkn-pac` and `opc` version `1.9.0` binaries for the supported platforms:
 
-* link:https://mirror.openshift.com/pub/openshift-v4/clients/pipeline/0.23.1/tkn-pac-linux-amd64-0.23.1.tar.gz[Linux (x86_64, amd64)]
-* link:https://mirror.openshift.com/pub/openshift-v4/clients/pipeline/0.23.1/tkn-pac-linux-s390x-0.23.1.tar.gz[Linux on IBM Z and LinuxONE (s390x)]
-* link:https://mirror.openshift.com/pub/openshift-v4/clients/pipeline/0.23.1/tkn-pac-linux-ppc64le-0.23.1.tar.gz[Linux on IBM Power Systems (ppc64le)]
-* link:https://mirror.openshift.com/pub/openshift-v4/clients/pipeline/0.23.1/tkn-pac-macos-amd64-0.23.1.tar.gz[Mac]
-* link:https://mirror.openshift.com/pub/openshift-v4/clients/pipeline/0.23.1/tkn-pac-windows-amd64-0.23.1.zip[Windows] 
+* link:https://mirror.openshift.com/pub/openshift-v4/clients/pipeline/1.9.0/tkn-linux-amd64.tar.gz[Linux (x86_64, amd64)]
+* link:https://mirror.openshift.com/pub/openshift-v4/clients/pipeline/1.9.0/tkn-linux-s390x.tar.gz[Linux on IBM Z and LinuxONE (s390x)]
+* link:https://mirror.openshift.com/pub/openshift-v4/clients/pipeline/1.9.0/tkn-linux-ppc64le.tar.gz[Linux on IBM Power (ppc64le)]
+* link:https://mirror.openshift.com/pub/openshift-v4/clients/pipeline/1.9.0/tkn-macos-amd64.tar.gz[macOS]
+* link:https://mirror.openshift.com/pub/openshift-v4/clients/pipeline/1.9.0/tkn-windows-amd64.zip[Windows]
 +
 [NOTE]
 ====
-The binaries are compatible with `tkn` version `0.23.1`.
+The binaries are compatible with `tkn` version `1.9.0`.
 ====
 
 // In addition, you can install `tkn-pac` using the following methods:
 
 // [CAUTION]
 // ====
-// The `tkn-pac` CLI tool available using these methods is _not updated regularly_. 
+// The `tkn-pac` CLI tool available using these methods is _not updated regularly_.
 // ====
 
 // * Install on Linux or Mac OS using the `brew` package manager:
```

Comment:
@vdemeester @chmouel does brew contain upstream or downstream binaries?

Reply:
I commented them out because these are not officially supported methods. Perhaps these lines can be safely deleted. 
When PAC was first released as TP, I wrote them because they were mentioned in upstream PAC documentation.

Reply:
Oh, I didn't notice that it's commented out. IMHO we can remove it